content/docs/join_us/recruitment_FAQs.md

```diff
@@ -241,6 +241,26 @@ This list is not exhaustive, but some examples include:
 - Knowledge sharing (organising our [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}))
 
+### Can you tell me more about what it is like working at The Alan Turing Institute?
+
+At The Turing, we want to make sure you feel comfortable and supported in how you work. Here’s a bit about what to expect:
+
+- **Hybrid Working**: We only ask that you come into the office four days a month — the rest is up to you! Some of the team work primarily remotely, while others come in every day. There is no pressure or expectation to be in the office more often than the minimum four days.
+
+- **Flexible Hours**: Our core hours are 10 am to 4 pm, but you’re free to work whenever works best for you. Whether you’re an early bird or prefer working later, we trust you to manage your time in a way that fits your life and responsibilities. We also understand that everyone has commitments — whether it’s a doctor’s appointment, picking up kids, or running errands — so taking time for those is completely fine.
+
+- **Work-Life Balance**: We value a healthy work-life balance and believe you should be able to switch off outside of work. There is no expectation to respond to emails or messages outside your working hours!
+
+- **Hot Desks**: We use hot desks, meaning you can sit wherever you like when you’re in the office, on a first-come, first-served basis. The Research Engineering Group typically sits near one another on the first floor. If you need a permanent or bespoke workspace (such as standing/adjustable desks, footrests, or ergonomic keyboards), we can work with the facilities team to make it happen.
+
+- **Meeting Etiquette**: We typically like to have our cameras on during Zoom meetings, but it’s absolutely fine to have your camera off if you feel more comfortable. And, while we encourage you to take regular breaks as needed, it’s also ok to eat or snack during meetings — in fact, we host regular lunchtime tech talks or reading groups where the team can lunch and learn simultaneously.
+
+- **Come as You Are**: We don’t have a dress code, so you can dress in whatever makes you feel comfortable. Just keep it professional when needed!
+
+- **Office Snacks and Team Lunches**: We love having team lunches together when we’re in. There is also free fruit, biscuits, juice, tea, and coffee to keep you going throughout the day. The kitchen area is great for group coffee breaks, or you can snack at your desk anytime.
+
+- **Location and Socials**: Our offices are based in the British Library, right in the heart of London and just 5 minutes from King’s Cross Station. As part of the Knowledge Quarter, there’s always something exciting happening nearby. We also have plenty of social outings organised by and for Turing staff, including five-a-side football, tennis, climbing, Thursday pub club, French club, film club, hikes…the list goes on!
+
```

Comment:
```suggestion
- **Location and Socials**: Our offices are based in the British Library, right in the heart of London and just 5 minutes from King’s Cross Station. As part of the Knowledge Quarter, there’s always something exciting happening nearby. We also have plenty of social outings organised by and for Turing staff, including five-a-side football, tennis, climbing, Thursday pub club, French club, film club, hikes…the list goes on (although absolutely none of it is compulsory!).

```

As somebody who genuinely appreciated that "rest in your room" was an option for RSECon social.

Reply:
(It's not so much that I was going to actually do that, but just the messaging of 'it's okay to not be extroverted'.)